modules/cnf-topology-aware-lifecycle-operator-preparing-for-updates.adoc

```diff
@@ -0,0 +1,107 @@
+// Module included in the following assemblies:
+// Epic CNF-2600 (CNF-2133) (4.10), Story TELCODOCS-285
+// * scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="talo-platform-prepare-end-to-end_{context}"]
+= End-to-end procedures for updating clusters in a disconnected environment
+
+If you have deployed spoke clusters with distributed unit (DU) profiles using the GitOps ZTP with the {cgu-operator-first} pipeline described in "Deploying distributed units at scale in a disconnected environment", this procedure describes how to upgrade your spoke clusters and Operators.
+
+[id="talo-platform-prepare-for-update_{context}"]
+== Preparing for the updates
+
+If both the hub and the spoke clusters are running {product-title} 4.9, you must update ZTP from version 4.9 to 4.10. If {product-title} 4.10 is used, you can set up the environment.
```

Comment:
Sentence 
"If OpenShift Container Platform 4.10 is used, you can set up the environment." 
is not clear to me.
 OCP 4.10 where? Spoke? Hub?
Also "set up the environment" is confusing. How about "proceed to next section"?